### DISCUSSION TITLE
English OCR

### DISCUSSION THREAD

**Discussion by Venator-86th** [VoxelCubes/PanelCleaner#57] [Q&A] [answered]
Is it possible to use OCR for English text? In the settings, I saw that other models can be used, but I do not know what exactly I need to look for and how to connect them to the program. I tried to use the default model for English text, it does a good job, but maybe there is a way to improve the result?

**Comment by VoxelCubes** [comment, marked as answer]
The OCR engine is mangaocr, it only really supports japanese text. I may try offering tesseract as well some time, if it performs well, which supports nearly any language. Downside is that it wasn't meant to be used for manga, so it fails at vertical text and strange fonts.

The model path is for the text detector. Should there ever be a new comictextdetector model that's compatible, you could just plug it in yourself, if I don't update it to downloading it by default. That unfortunately is separate from OCR and thus won't help in your situation. I'll note it for the future though.

**Reply by VoxelCubes** [reply]
We're currently working on adding tesseract support, see #85 for details, hopefully that will serve your needs when it's ready.
